_maps/ringworm.json

```diff
@@ -0,0 +1,30 @@
+{
+	"version": 1,
+	"map_name": "ringworm",
```

Comment:
you called it RingWorm in `area_spawn_subsystem` and that's why you failing area_spawn unit tests (well, that's not THE reason, but i think you placed all skyrat stuff yourself, right?)

Reply:
Yeah, I had issues with a previous linter due to the map folder being capitalized (for some god-forsaken reason??) and forgot to edit this one, thanks for the catch!